src/sentry/integrations/slack/message_builder/notifications/issues.py

```diff
@@ -45,12 +48,13 @@ def __init__(
 
     def build(self) -> SlackBody:
         group = getattr(self.notification, "group", None)
-        return SlackIssuesMessageBuilder(
+        return SlackReleaseIssuesMessageBuilder(
             group=group,
             event=getattr(self.notification, "event", None),
             tags=self.context.get("tags", None),
             rules=getattr(self.notification, "rules", None),
             issue_details=True,
             notification=self.notification,
             recipient=self.recipient,
+            release_version=getattr(self.notification, "release_version", None),
```

Comment:
this assumes we've attached a release_version to the notification object @barkbarkimashark